File: templates/variants/go-chi/internal/temporal/workflows.go
package temporalapp

import (
	"time"

	"go.temporal.io/sdk/workflow"
)

func WaitlistFollowUpWorkflow(ctx workflow.Context, input WaitlistFollowUpInput) (WaitlistFollowUpResult, error) {
	options := workflow.ActivityOptions{
		StartToCloseTimeout: time.Minute,
	}
	ctx = workflow.WithActivityOptions(ctx, options)

	var result WaitlistFollowUpResult
	err := workflow.ExecuteActivity(ctx, "RecordWaitlistFollowUp", input).Get(ctx, &result)
	return result, err
}

File: templates/variants/bun-connectrpc/src/temporal/workflows.ts
import { proxyActivities } from "@temporalio/workflow";
import type * as activities from "./activities";

const { recordWaitlistFollowUp } = proxyActivities<typeof activities>({
  startToCloseTimeout: "1 minute",
});

export async function waitlistFollowUpWorkflow(input: activities.WaitlistFollowUpInput) {
  return await recordWaitlistFollowUp(input);
}
